Server/DiscordBot.cs

```diff
@@ -88,9 +98,28 @@ private static List<string> SplitMessage(string message, int maxSizePerElem = 20
             Logger.Info(
                 $"Discord bot logged in as {DiscordClient.CurrentUser.Username}#{DiscordClient.CurrentUser.Discriminator}");
             Reconnecting = false;
-            string mentionPrefix = $"{DiscordClient.CurrentUser.Mention} ";
+            string mentionPrefix = $"{DiscordClient.CurrentUser.Mention}";
             DiscordClient.MessageCreated += async (_, args) => {
-                if (args.Author.IsCurrent) return;
+                if (args.Author.IsCurrent) return; //dont respond to commands from ourselves (prevent "sql-injection" esq attacks)
+                //prevent commands via dm and non-public channels
+                if (Config.CommandChannel == null) {
+                    if (!warnedAboutNullLogChannel) {
+                        Logger.Warn("You probably should set your CommandChannel in settings.json");
+                        warnedAboutNullLogChannel = true;
+                    }
```

Comment:
Generally warn on startup, not after. Server hosts probably won't see it.

Reply:
now in ctor of discordbot